modules/ROOT/pages/producing-ign.adoc

```diff
@@ -190,7 +190,7 @@ TIP: YAML files must have consistent indentation. Although Butane checks for syn
 +
 [source,bash]
 ----
-butane --pretty --strict < example.bu > example.ign
+butane --pretty --strict example.bu > example.ign
```

Comment:
Note this is now reading from `$PWD` being mounted into the container, whereas before this was handled by bash. It's fine in this simple example, but e.g. `butane --pretty --strict /some/other/path/example.bu > example.ign` won't necessarily work.

Overall, probably safer to just leave it in?

Reply:
Huh … now that's a tricky one. I know that running butane from a container is supported and works – personally I just never do it and instead use a local butane binary 🤷‍♂️. I am not even sure outside of the explicit container example on the same page the docs actually refer to this use case a lot/at all.

Reply:
OK the example in that section already talks about using stdin vs passing as a file, which is nice. I'm cool with this change then.